wix/main.wxs

```diff
@@ -62,16 +59,18 @@
                 <Directory Id='INSTALLDIR' Name='Volta'>
                 </Directory>
             </Directory>
+            <?if $(sys.BUILDARCH) = x64 ?>
             <Merge Id='VCRedist' SourceFile='wix\Microsoft_VC140_CRT_x64.msm' DiskId='1' Language='0'/>
+            <?endif ?>
```

Comment:
Why are you dropping this check?

Reply:
Since the VC++ redistribution runtime was only available for x64, I have added a condition so that it is only included for x64 as before.